### PR TITLE
Sprite as widget

### PR DESCRIPTION
**Pull request by erickzanardo**
I found no gain creating something similar for animations, what we have right now seems to be the best for animations already.

But for Sprites, I found that this is a good solution.

TODO: Still need to update docs.

### CODE REVIEW

File: lib/util.dart [hunk 1]
```diff
@@ -88,4 +89,27 @@ class Util {
...
+
+  @override
+  void paint(Canvas canvas, Size size) {
+    if (_sprite.loaded()) {
+      _sprite.render(canvas, size.width, size.height);
+    }
+  }
+
+  @override
+  bool shouldRepaint(widgets.CustomPainter old) => false;
```
Comment (luanpotter) [inline lib/util.dart]: if shouldRepaint returns false always, _sprite should be final... what do you think?

Comment (erickzanardo) [reply lib/util.dart]: Sounds good, I will change that later

Comment (erickzanardo) [reply lib/util.dart]: Done!

**Comment by erickzanardo** [comment]
I have also added some info on docs about sprite and animations as widgets.

**Pull request merged**